portal-api/src/v1/section-currency/index.js

```diff
@@ -1,15 +1,24 @@
 const { findOneCurrency } = require('../controllers/currencies.controller');
 
+/**
+ * Find a currency by currency code and return a simple object
+ * @param currencyCode - The currency code
+ * @returns {Promise<Object>} Empty object or object with currency properties
+ */
 const getCurrencyObject = async (currencyCode) => {
   const { data } = await findOneCurrency(currencyCode);
 
-  const { text, id, currencyId } = data;
+  if (data) {
+    const { text, id, currencyId } = data;
 
-  return {
-    text,
-    id,
-    currencyId,
-  };
+    return {
+      text,
+      id,
+      currencyId,
+    };
+  }
+
+  return {};
```

Comment:
After GOV design update - not sure why exactly, but this was erroring and causing a crash if no currency is found via `findOneCurrency` (DTFS2-7639, DTFS2-7640)

Therefore i've updated this to return an empty object.